tools/failingTests/fail.tests.ps1

```diff
@@ -0,0 +1,5 @@
+Describe "Failing test used to test CI Scripts" {
+    It "Should fail" {
+        1 | should be 2
+    }
+}
```

Comment:
I'll add the line ending later.  I'll use this change to remove the tag that forces a test failure.